### DISCUSSION TITLE
Suggestion: Sidebar for quick navigation

### DISCUSSION THREAD

**Discussion by NotAnotherGlitch** [standardnotes/forum#2538] [Suggestions]
I have recently thought about a possible new implementation for Standard Notes. Here is my idea:

With some selected text you are able to create a _'marker'_ point where the note will mark a **guidance jump**. This jump pulls the note to the section where the selected _'marker'_ point is present. This implementation would create an easier navigation for documents such as a password document where you might label a section of passwords based on their accounts. For example a social media section would have a selected _'marker'_ while a shopping section would have a different _'marker'_ allowing the user to switch quickly between multiple sections without having to _**tediously**_ scroll. 

Please consider this suggestion.

**Comment by atmoio** [comment]
It's a neat idea, but this belongs to the domain of editor extensions and not the core app. The core app is closed for new features in general. Anything new we add is likely going to be an extension. As for what's fair game, this would definitely fit, but it would probably be for a third-party developer to build this extension, at least as far as I can see.
